app/views/hosts/_list.html.erb

```diff
@@ -10,7 +10,7 @@
       <th class="hidden-tablet hidden-xs" width="10%"><%= sort :model, :as => _('Model') %></th>
       <th class="hidden-tablet hidden-xs" width="15%"><%= sort :hostgroup, :as => _("Host group") %></th>
       <th class="hidden-tablet hidden-xs" width="10%"><%= sort :last_report, :as => _('Last report'), :default => 'DESC' %></th>
-      <th ><%= _('Actions') %></th>
+      <th width="100px"><%= _('Actions') %></th>
```

Comment:
What you want to accomplish by using fixed-size width is that other buttons never cause this one to overflow, right?